docs/source/tutorials/e2e_flow.rst

```diff
@@ -319,126 +319,9 @@ Bay Area!
 Speeding up Generation using Quantization
 -----------------------------------------
 
-We saw that the generation recipe took around 11.6 seconds to generate 300 tokens.
-One technique commonly used to speed up inference is quantization. torchtune provides
-an integration with the `TorchAO <https://github.com/pytorch-labs/ao>`_
-quantization APIs. Let's first quantize the model using 4-bit weights-only quantization
-and see if this improves generation speed.
+We rely on `TorchAO <https://github.com/pytorch-labs/ao>`_ for `post training quantization <https://github.com/pytorch/ao/tree/main/torchao/quantization#quantization>`_, please take the fine tuned model from torchtune and quantize them directly in `TorchAO`, you can also run `eval/benchmark <https://github.com/pytorch/ao/tree/main/torchao/_models/llama>`_. in torchao as well.
```

Comment:
Can we include the few lines of code needed to do this with AO? Would make our tutorial way more clear.

Reply:
added